yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -1626,10 +2042,41 @@ module vyatta-protocols-frr-bgp-v1 {
 						uses parameters-distance;
 						uses parameters-dampening;
 					}
+
+					container label {
+						description "label value for VRF";
+						configd:help "label value for VRF";
+						container vpn {
+							description "Between current address-family and vpn";
+							configd:help "Between current address-family and vpn";
+							leaf export {
+								description "For routes leaked from current address-family to vpn";
+								configd:help "For routes leaked from current address-family to vpn";
+								type union {
+									type uint32 {
```

Comment:
I don't think setting a static vpn label is necessary. I don't think this should be exposed in the yang.